app/classifier/classifier.jsx

```diff
@@ -175,7 +172,7 @@ class Classifier extends React.Component {
       }, false);
     }
 
-    if (!isInProgress) {
+    if (this.props.feedback.active && !isInProgress) {
```

Comment:
I'd recommend putting this check at the top of the function and returning early if false - it gets the 'no feedback' case out of the way quicker.

Reply:
Good point.

Reply:
Fixed in 03ce256.